pkgs/development/python-modules/hatch-requirements-txt/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, hatchling
+}:
+
+buildPythonPackage rec {
+  pname = "hatch-requirements-txt";
+  version = "0.2.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.6.1";
+
+  # We use the Pypi release, as it provides prebuild webui assets,
+  # and its releases are also more frequent than github tags
+  src = fetchPypi {
+    inherit version;
+    pname = builtins.replaceStrings ["-"] ["_"] pname;
+    hash = "sha256-I8z34vvPK7Mg9+Xg2nMgNcgeh5QFB0LV2j0iwzA1QGc=";
+  };
+
+  propagatedBuildInputs = [
+    hatchling
+  ];
+
+  pythonImportsCheck = [
+    (builtins.replaceStrings ["-"] ["_"] pname)
```

Comment:
For sake of consistency I'd rather pick one of the two suggested fixes than to do two different things. Which one do you prefer?

Reply:
Which suggested fixes you mean?

We never substitute pythonImportsCheck because it is pure coincidence that it matches pname at all and things likely break when normalizing pname.